.github/workflows/build-targets.yml

```diff
@@ -0,0 +1,52 @@
+name: Build ML examples
+on:
+  workflow_dispatch:
+  pull_request:
+    branches: [main]
+  push:
+    branches: [main]
+  schedule:
+    - cron: '30 22 28 * 0'
+
+jobs:
+  build-target:    
+    name: Build target
+
+    strategy:
+      fail-fast: true
+      matrix:
+        context: [object-detection.Debug+Alif-E7-M55-HP, object-detection.Debug+AVH-SSE-300-U55, kws.Debug+Alif-E7-M55-HE, kws.Debug+AVH-SSE-310-U65, kws.Debug+FRDM-K64F, kws.Debug+STM32F746-DISCO]
```

Comment:
Any reason for not building the release contexts?

Reply:
No particular reason. Should I add both release and debug contexts?